llvm/cmake/modules/FindGRPC.cmake

```diff
@@ -87,6 +87,10 @@ else()
     message(STATUS "Using grpc++: " ${GRPC_LIBRARY})
     set_target_properties(grpc++ PROPERTIES IMPORTED_LOCATION ${GRPC_LIBRARY})
     target_include_directories(grpc++ INTERFACE ${GRPC_INCLUDE_PATHS})
+    find_library(GPR_LIBRARY gpr ${GRPC_OPTS} REQUIRED)
```

Comment:
What is GPR? Why do we need it?

Reply:
This is `libgpr.dylib`, part of the set of `gRPC` libraries; I get linker errors for missing symbols `_gpr_log` and `_gpr_unreachable_code`  when linking `llvm-remote-cache-test` without this change. I intend to push this change to upstream as well.